### ISSUE TITLE
Incorrect mailbox setting not being read from config

### ISSUE THREAD

**Issue by matthewblott** [neomutt/neomutt#4050]
I'm presented with the following at the bottom of the screen when I run `neomutt`:

```
Unable to open mailbox /var/mail/Matt
```

The `folder` setting in `.muttrc` was actually set to `~/.local/share/mail/` so it doesn't appear to be picking up the correct value from the configuration file.

I'm using M1 on macOS with Proton Mail Bridge. I was mostly following [this](https://spaceandtim.es/code/protonmail_mutt/) tutorial I found online which seems to work for everyone.

Things I've tried:
- Installed Thunderbird to confirm Proton Mail Bridge works.
- Created a folder at `/var/mail/Matt` with the correct permissions and changed the setting in `.muttrc` to `set folder=/var/mail/Matt`.
- Changed the setting in `.muttrc` to `set folder="imap://$my_user:$my_pass@127.0.0.1:1143"`.

None of these things have worked thus far. Any help appreciated :-)

**Comment by flatcap** [comment]
Try something like:

```sh
# Tell NeoMutt which folder to open first
set spool_file = "+inbox"
```

When NeoMutt starts it wants to open a folder.
If you don't tell it which, then it'll try `$MAIL` from your environment or `/var/mail/USERNAME`

**Comment by matthewblott** [comment]
Thanks, I tried but it didn't make any difference (I tried a few times and placing it at different points in the file). The result is the same with the same message at the foot of the terminal window.

**Comment by matthewblott** [comment]
Thanks for the quick response. I tried everything you suggested. I tried removing the `.offlineimaprc` file to see if it made any difference (it didn't).

Here's my `.muttrc` file:

```
set my_user=matt@example.com
set my_pass=proton-bridge-password

set spool_file = "~/.mail"
set mbox_type=Maildir
set folder="~/.mail"
# I tried with this setting as well ...
# set folder="imap://$my_user:$my_pass@127.0.0.1:1143"
set record=+Sent
set postponed=+Drafts
set trash=+Trash
set mail_check=2 # seconds

# smtp
set smtp_url=smtp://$my_user:$my_pass@127.0.0.1:1025
set ssl_force_tls
set ssl_starttls
```

Here's the contents of my `.offlineimaprc` file:

```
[general]
accounts = main

[Account main]
localrepository = main-local
remoterepository = main-remote
autorefresh = 0.2
quick = 10
postsynchook = notmuch new

[Repository main-local]
type = Maildir
localfolders = "~/.mail"
sync_deletes = yes

[Repository main-remote]
type = IMAP
remoteport = 1143
remotehost = 127.0.0.1
remoteuser = matt@example.com
remotepass = proton-bridge-password
keepalive = 60
holdconnectionopen = yes
expunge = yes
folderfilter = lambda foldername: foldername in ['INBOX', 'Archive', 'Sent']
ssl = no
```

The settings above are copied directly from the tutorial I refer to in the opening post.

**Comment by matthewblott** [comment]
Okay, I'm in business (well, I can at least login and view email). As usual it turned out to be something tiny. I thought it was close and hunted everywhere to see what configurations people were using. It's very hard to find anyone using macOS *and* ProtonMail *and* Neomutt, I was always finding 2 out of 3. Anyway I checked out [this](https://gist.github.com/syncom/6b77792adb59ccdaa5d36b30273abc33) setup (using Gmail) and wondered what the folders were set to. The offending setting was ```spoolfile``` and so this is now my *full* configuration file: 

```
set my_user=matt@example.com
set my_pass=proton-bridge-password
set mbox_type=Maildir
set folder="imap://$my_user:$my_pass@127.0.0.1:1143"
set spoolfile = "+INBOX"
set smtp_url=smtp://$my_user:$my_pass@127.0.0.1:1025
```

The above is pretty sparse (this is also without using the `offlineimaprc` settings). I haven't tried using anything yet but I've got past the main hurdle and the reason for this ticket so I'm going to close it. Hopefully this will help others - it is fairly straightforward to connect if you know how! Cheers :-)
